src/Spectre.libGenetic.Tests/GeneticAlgorithmTest.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+* GeneticAlgorithm.cpp
+* Tests general-purpose genetic algorithm.
+*
+Copyright 2017 Grzegorz Mrukwa
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#define GTEST_LANG_CXX11 1
+
+#include <gtest/gtest.h>
+
+namespace
+{
+TEST(GeneticAlgorithmTest, dummy)
```

Comment:
These are placeholders, to not to forget of real tests, which should be introduced there.